src/main/kotlin/com/dansplugins/factionsystem/locks/MfLockService.kt

```diff
@@ -43,6 +53,39 @@ class MfLockService(private val plugin: MedievalFactions, private val repository
         ServiceFailure(exception.toServiceFailureType(), "Service error: ${exception.message}", exception)
     }
 
+    fun unlock(block: Block): Int {
```

Comment:
Can we give this a more informative type rather than an int? We should avoid magic numbers where possible

Reply:
Yup! I've created and utilized the 'MfUnlockResult' enumeration instead.

Reply:
The push failed actually hold on I need to reinstall java.

Reply:
Okay changes have been pushed.